modules/pubmaticBidAdapter.js

```diff
@@ -601,13 +601,20 @@ function _handleDigitrustId(eids) {
   }
 }
 
-function _handleTTDId(eids) {
+function _handleTTDId(eids, validBidRequests) {
+  let ttdId = null;
   let adsrvrOrgId = config.getConfig('adsrvrOrgId');
-  if (adsrvrOrgId && utils.isStr(adsrvrOrgId.TDID)) {
+  if (validBidRequests[0] && validBidRequests[0].userId && utils.isStr(validBidRequests[0].userId.tdid)) {
```

Comment:
Could possibly use utils.deepAccess here instead:

`if (isStr(deepAccess(validBidRequests, '0.userId.tdid'))) {` 
instead of
`if (validBidRequests[0] && validBidRequests[0].userId && utils.isStr(validBidRequests[0].userId.tdid)) {`

Reply:
Won't request changes because of this,

But will wait to merge in case the change is wanted to be made!

Don't worry we will get it in before the release!

Reply:
Sure @robertrmartinez , thank you for this suggestion :)